src/OfficeToolPlus/Dictionaries/Languages/pt-br.xaml

```diff
@@ -573,9 +573,9 @@
     <sys:String x:Key="Hash_Checker_Export.AsMD">Exportar como _Markdown</sys:String>
     <sys:String x:Key="Hash_Checker_Export.AsJSON">Exportar como _JSON</sys:String>
     <sys:String x:Key="Hash_Checker_Compare">Compare</sys:String>
```

Comment:
Is that missing? `Compare` -> `Comparar`?

Reply:
Yes, I forgot this one. Already fixed.